templates/blog/_newsletter-signup-form.html

```diff
@@ -0,0 +1,27 @@
+<div class="p-card--newsletter">
+  <p class="p-heading--four">
+    Subscribe to our weekly newsletter
+  </p>
+  <form action="https://pages.canonical.com/index.php/leadCapture/save" method="post" id="mktoForm_3415">
+    <div>
+      <label class="u-off-screen" for="Email">Work email:*</label>
+      <input class="u-no-margin--bottom" id="Email" placeholder="Work email" name="Email" maxlength="255" type="email" required />
+    </div>
+    <p>
+      <small>In submitting this form, I confirm that I have read and agree to <a href="/legal/data-privacy/newsletter">Canonical’s Privacy Notice</a> and <a href="/legal/data-privacy">Privacy Policy</a>.</small>
+    </p>
+    <p class="u-no-margin--bottom">
+      <input type="hidden" name="Consent_to_Processing__c" value="Yes" />
+      <button class="p-button" type="submit">Subscribe now</button>
+    </p>
+    <input type="hidden" name="formid" value="3415" />
```

Comment:
Is this the correct formid

Reply:
Yep, @caldav set a new one up for us and passed the ID on.